android-runtime/runtime/src/main/java/io/sdkgen/runtime/SdkgenHttpClient.kt

```diff
@@ -252,6 +269,7 @@ open class SdkgenHttpClient(
                 addProperty("name", functionName)
                 add("args", bodyArgs ?: JsonObject())
                 add("deviceInfo", makeDeviceObj())
+                add("extras",  createJsonObjectFromExtrasMap())
```

Comment:
Com base em https://stackoverflow.com/a/44824039, aparentemente esse método poderia ser substituído por:

```suggestion
                add("extras",  gson.toJsonTree(extras).getAsJsonObject())
```